packages/tx-construction/src/tx-builder/initializeTx.ts

```diff
@@ -15,12 +14,11 @@ const dRepPublicKeyHash = async (addressManager?: Bip32Account): Promise<Ed25519
 
 const DREP_REG_REQUIRED_PROTOCOL_VERSION = 10;
 
-const isActive = (drepDelegatee?: Cardano.DRepDelegatee): boolean => {
-  const drep = drepDelegatee?.delegateRepresentative;
-  if (!drep || (Cardano.isDrepInfo(drep) && drep.active === false)) {
-    return false;
-  }
-  return true;
+const hasVoteDelegated = ({ dRepDelegatee }: RewardAccountWithPoolId): boolean => {
```

Comment:
Please reduce the type to what the function requires, so callers don't have to provide more than what's necessary:
```diff
- const hasVoteDelegated = ({ dRepDelegatee }: RewardAccountWithPoolId): boolean => {
+ const hasVoteDelegated = ({ dRepDelegatee }: Pick<RewardAccountWithPoolId, 'drepDelegatee'>): boolean => {
```